### ISSUE TITLE
Error 206 triggered with nested literal brace

### ISSUE THREAD

**Issue by OrangeDog** [warpnet/salt-lint#257]
**Describe the bug**
Error 206 is erroneously reported for the following construction

**To Reproduce**
```
[206] [LOW] Jinja variables should have spaces before and after: '{{ var_name }}'
test.sls:3
    - name: '{${{ key }}}'
```

**Expected behavior**
No issue on this line.

**Desktop (please complete the following information):**

- OS: Windows 10
- Version 0.6.1

**Comment by mjtrangoni** [comment]
@OrangeDog as a workaround, you can add # noqa 206 at the end of the line.

**Comment by jbouter** [comment]
Hi @OrangeDog 👋🏻 

Thanks for your report! We'll work on this when we can. In the meantime, you can work around this issue using @mjtrangoni's suggestion
